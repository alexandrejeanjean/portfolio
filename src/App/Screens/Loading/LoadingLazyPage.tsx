import React from 'react'
// import BubbleT from '../../Assets/BubbleT.svg'
import '../Home/home.scss'

const Loading = () => {
  return (
    <>
      <main className='home-wrapper page'>
        <section>
          <div className='home-title-wrapper'>
            <div className='home-title-content'>
              <h1 className='home-title'>Chargement de l'itin√©raire...</h1>
            </div>
          </div>
          <div className='home-bubble-wrapper'>
            {/* <img src={BubbleT} className='home-bubble' alt='' /> */}
          </div>
        </section>
      </main>
    </>
  )
}

export default Loading
