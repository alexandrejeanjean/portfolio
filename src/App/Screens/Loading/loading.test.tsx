import React from "react";
import { shallow } from "enzyme";
import LoadingLazyPage from "./LoadingLazyPage";

let wrapper: any;

beforeEach(() => {
  wrapper = shallow(<LoadingLazyPage />);
});

describe("LoadingLazyPage view", () => {
  it("should render LoadingLazyPage component correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
