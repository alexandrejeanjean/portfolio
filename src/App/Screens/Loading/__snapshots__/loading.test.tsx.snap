// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingLazyPage view should render LoadingLazyPage component correctly 1`] = `
<Fragment>
  <main
    className="loader-wrapper page"
  >
    <section
      className="loader"
    >
      <Loader
        className=""
        color="rgba(137, 194, 217, 0.3)"
        height={100}
        style={Object {}}
        timeout={3000}
        type="Circles"
        visible={true}
        width={100}
      />
    </section>
  </main>
</Fragment>
`;
